packages/create-keystone-app/example-projects/starter/initial-data.js

```diff
@@ -0,0 +1,44 @@
+const crypto = require('crypto');
+const randomString = () => crypto.randomBytes(6).hexSlice();
+
+module.exports = async keystone => {
+  // Count existing users
+  const {
+    data: {
+      _allUsersMeta: { count },
+    },
+  } = await keystone.executeQuery(
+    `query {
+      _allUsersMeta {
+        count
+      }
+    }`
+  );
+
+  if (count === 0) {
+    const password = randomString();
+    const email = 'admin@keystonejs.com';
```

Comment:
Can we make this `@example.com` to avoid any issues with people in the future accidentally sending emails, etc?